scss/styles.scss

```diff
@@ -1,3 +1,5 @@
 .body {
```

Comment:
I think you dont need to style the body tag. Style the main container instead. :dancing_women: